.ci/bump-version.sh

```diff
@@ -13,6 +13,9 @@ docker run --rm -t \
   mcr.microsoft.com/dotnet/core/sdk:2.2 /bin/sh -c "
     dotnet add package Elastic.Apm.NetCoreAll -v ${AGENT_VERSION}"
 
+## Bump agent version in the Dockerfile
+sed -ibck "s#\(org.label-schema.version=\)\(.*\)#\1\"${AGENT_VERSION}\"#g" Dockerfile
```

Comment:
```suggestion
sed -ibck "s#\(org.label-schema.version=\)\(\".*\"\)\(.*\)#\1\"${AGENT_VERSION}\"\3#g" Dockerfile
```

Reply:
Otherwise:
```diff
diff --git a/Dockerfile b/Dockerfile
index 7cd2f52..368f316 100755
--- a/Dockerfile
+++ b/Dockerfile
@@ -39,7 +39,7 @@ LABEL \
     org.label-schema.schema-version="1.0" \
     org.label-schema.vendor="Elastic" \
     org.label-schema.name="opbeans-python" \
-    org.label-schema.version="5.7.0" \
+    org.label-schema.version="5.8.0"
```

and now:

```diff
diff --git a/Dockerfile b/Dockerfile
index 7cd2f52..368f316 100755
--- a/Dockerfile
+++ b/Dockerfile
@@ -39,7 +39,7 @@ LABEL \
     org.label-schema.schema-version="1.0" \
     org.label-schema.vendor="Elastic" \
     org.label-schema.name="opbeans-python" \
-    org.label-schema.version="5.7.0" \
+    org.label-schema.version="5.8.0" \

```